lib/validation.ts

```diff
@@ -2,6 +2,38 @@
 import { z } from "zod";
 
 export const LoginValidationSchema = z.object({
-    email: z.string().email('Email no valido'),
-    password: z.string().min(8, { message: 'El password debe ser mínimo de 8 caracteres.' }),
-});
\ No newline at end of file
+  email: z.string().email("Email no valido"),
+  password: z
+    .string()
+    .min(8, { message: "El password debe ser mínimo de 8 caracteres." }),
+});
+
+export const RegistrationValidationSchema = z
+  .object({
+    email: z.string().email("Email no válido"),
+    password: z
+      .string()
+      .min(8, { message: "El password debe ser mínimo de 8 caracteres." }),
+    confirmPassword: z.string(),
+    name: z
+      .string()
+      .min(2, { message: "El nombre debe contener mínimo 2 caracteres." })
+      .max(50, { message: "El nombre debe contener maximo 50 caracteres." }),
+    username: z
+      .string()
+      .min(2, {
+        message: "El nombre de usuario debe contener mínimo 2 caracteres.",
+      })
+      .max(50, {
+        message: "El nombre de usuario debe contener maximo 2 caracteres.",
+      }),
+    checkbox: z.boolean().default(false).optional(),
```

Comment:
same way here. sort fields in the same order that register form inputs

Reply:
solved